src/components/MetricsPane/MetricsPane.jsx

```diff
@@ -65,6 +70,8 @@ const MetricsPane = ({ showMetricsPane, setShowMetricsPane, showLoadingIndicator
   } = useFilterProjectsContext()
   const [selectedSampleEvent, setSelectedSampleEvent] = useState(null)
   const [metricsView, setMetricsView] = useState('summary')
+  const isMapView = view === 'mapView'
+  const map = useMap()[MAIN_MAP_ID] // the docs for react-map-gl are not clear on the return object for useMap. This is what works in testing. Further, its not a 'ref' its the actual map instance.
```

Comment:
This is where we grab the map instance 'provided' for earlier. This isnt the typical syntax, but the way this third party library exposes the context. 


The 'React' way of getting context values is `const {value1, value2, ...} = useContext(WhateverThecontextIsNamed)` although its evolved over time so you will see some other ways as well.